test/unit/ndr_error/log_test.rb

```diff
@@ -222,14 +217,9 @@ def setup
     end
 
     test 'should store the hostname if configured to do so' do
-      begin
-        original_hostname_identifier = NdrError.hostname_identifier
-        NdrError.hostname_identifier = -> { 'Dummy Host' }
-
+      with_config(:hostname_identifier, -> { 'Dummy Host' }) do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.